tests/sentry/api/endpoints/test_user_avatar.py

```diff
@@ -164,7 +162,6 @@ def test_put_upload_saves_to_control_file(self):
         assert isinstance(avatar.get_file(), ControlFile)
         assert ControlFile.objects.filter(id=avatar.control_file_id).exists()
 
-    @pytest.mark.skipif(use_split_dbs(), reason="requires single/monolith db mode")
     @mock.patch("sentry.tasks.files.copy_file_to_control_and_update_model.apply_async")
     def test_get_upload_use_control_during_update(self, mock_task):
```

Comment:
This test could be removed now. We don't need to test the intermediary step anymore.